jobs/aws/eks-anywhere-build-tooling/eks-anywhere-attribution-periodics-release-0.15.yaml

```diff
@@ -19,7 +19,7 @@
 ################################################################################
 
 periodics:
-- name: eks-anywhere-attribution-periodic-release-0-14
+- name: eks-anywhere-attribution-periodic-release-0-15
```

Comment:
Referencing the last two related PRs, it seems as though that is the case
- https://github.com/aws/eks-anywhere-prow-jobs/pull/249
- https://github.com/aws/eks-anywhere-prow-jobs/pull/235

Additionally the periodic checksum files have been removed since
- https://github.com/aws/eks-anywhere-prow-jobs/pull/265